src/service/viewport-impl.js

```diff
@@ -934,16 +861,20 @@ export class ViewportBindingNaturalIosEmbed_ {
 
   /** @override */
   getLayoutRect(el) {
-    const b = el./*OK*/getBoundingClientRect();
-    return layoutRectLtwh(Math.round(b.left + this.pos_.x),
-        Math.round(b.top + this.pos_.y),
-        Math.round(b.width),
-        Math.round(b.height));
+    return super.getLayoutRect(el, this.pos_.x, this.pos_.y);
   }
 
   /** @override */
-  setScrollTop(scrollTop) {
-    this.setScrollPos_(scrollTop || 1);
+  setScrollTop(scrollPos) {
+    // Stop us from scrolling _all_ the way to the top and triggering an
+    // annoying bounce animation.
+    scrollPos = Math.max(scrollPos, 1);
+    if (!this.scrollMoveEl_) {
+      return;
+    }
+    scrollPos -= this.getScrollTop();
```

Comment:
This one is not quiet clear. Why do we have a relative expression when setting an absolute value?